src/main/java/seedu/address/logic/commands/client/EditClientCommand.java

```diff
@@ -14,10 +25,55 @@ public class EditClientCommand extends ClientCommand {
 
     public static final String MESSAGE_SUCCESS = "Edited Client: %1$s";
 
-    // TODO: implement
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits a project in the address book. "
+            + "Parameters: "
+            + PREFIX_NAME + "NAME "
+            + PREFIX_CLIENT_EMAIL + "CLIENT EMAIL "
+            + PREFIX_CLIENT_PHONE + "PHONE "
+            + PREFIX_CLIENT_ID + "CLIENT ID "
+            + "Example: " + COMMAND_WORD + " "
+            + COMMAND_FLAG + " "
+            + PREFIX_NAME + "John "
+            + PREFIX_CLIENT_EMAIL + "john@gmail.com "
+            + PREFIX_CLIENT_PHONE + "12345678 "
```

Comment:
Would be good to have a more realistic number, or something that starts with 9

Reply:
Thank you for the suggestion! However, it is not uncommon for the operational phone numbers to be denoted by 12345678 and its variants, particularly in official guides (so that we don't accidentally use an actual phone number).